exporter/awskinesisexporter/internal/batch/aggregator.go

```diff
@@ -0,0 +1,75 @@
+package batch
+
+import (
+	"crypto/md5"
+
+	k "github.com/aws/aws-sdk-go/service/kinesis"
+	"google.golang.org/protobuf/proto"
+)
+
+var (
+	magicNumber           = []byte{0xF3, 0x89, 0x9A, 0xC2}
+	partitionKeyIndexSize = 8
+	maxAggregationCount   = 4294967295
```

Comment:
Can I ask where this value comes from?

Reply:
https://github.com/awslabs/amazon-kinesis-producer/blob/master/aggregation-format.md

Reply:
maybe we should include a copy of this readme or at least add a reference to it in the docs.